DataConverter/Catalyst.DataConverter/HierarchicalDataTransformerHelper.cs

```diff
@@ -32,6 +34,22 @@ namespace DataConverter
     public class HierarchicalDataTransformerHelper : IHierarchicalDataTransformerHelper
     {
 
+        public async Task<JobData> TransformDataAsync(
```

Comment:
This will go away eventually, right?

Reply:
Yes.  Should have taken it out before pushing.  Good catch (it's gone now).